elastic/kibana
# [Fleet] Show callout when EPR unavailable

## Summary

Closes https://github.com/elastic/kibana/issues/115564

-----

Show callout on the integration page when EPR is unavailable.

![image](https://user-images.githubusercontent.com/1833023/140418726-2bb74ca2-f429-4201-804a-07d2d48133a2.png)

-----

How to test:
- make sure EPR is unavailble. Easiest way to do this is by add an entry to your `etc/host` file for `https://epr-snapshot.elastic.co` to loopback when testing this locally from `main` branch. (e.g. ```127.0.0.1       epr-snapshot.elastic.co```
)
- navigate to Integrations page



### Checklist

Delete any items that are not applicable to this PR.
